lib/glific/stats.ex

```diff
@@ -449,4 +449,26 @@ defmodule Glific.Stats do
     })
     |> Repo.one()
   end
+
+  @doc """
+  Get the details of the usage for this organization, from start_date to end_date both inclusive
+  """
+  @spec get_message_count(non_neg_integer(), Keyword.t()) :: {:ok, non_neg_integer()}
+  def get_message_count(organization_id, opts \\ []) do
+    default_start_date = Timex.shift(DateTime.utc_now(), days: -1)
+    start_date = Keyword.get(opts, :start_date, default_start_date)
+    end_date = Keyword.get(opts, :end_date, DateTime.utc_now())
+
+    %{messages: messages} =
+      Stat
+      |> where([s], s.organization_id == ^organization_id)
+      |> where([s], s.period == "hour")
```

Comment:
Can we send the period also as an argument? Just check my last comment.

Reply:
I have made requested changes